src/main/java/radiounit/ConcreteRadioUnit.java

```diff
@@ -170,7 +170,7 @@ public void removeAllCarriers() {
 		try {
 			state.removeAllCarriers();
 		} catch (IllegalStateTransitionException e) {
-			e.printStackTrace();
+			System.out.println(e.getMessage());
```

Comment:
Nothing wrong with this change, I'm just curious if you did it to hide implementation details? My understanding is this change doesn't print some information such as the line number where the error occured

Reply:
I changed it so the entire stack trace wouldn't print to the console (which appears in red and seems like a crash) whenever this illegal state change takes place. It was moreso done to improve the user experience of using the NMS. But this way we can show off this functionality to Ista and it won't fill up the console whenever it occurs. It will just print the useful message as to why the illegal state exception got thrown.